lib/IRGen/IRGenSIL.cpp

```diff
@@ -2487,6 +2538,12 @@ void IRGenModule::emitSILFunction(SILFunction *f) {
       f->isAvailableExternally())
     return;
 
+  // Type metadata for foreign references is not yet supported on Windows.
```

Comment:
Is there/should there be a tracking issue for this feature on Windows in the swiftlang/swift issue tracker?

Reply:
@egorzhdan do you know if we already track this somewhere?

Reply:
I don't think we have an issue yet, but it would make sense to have one! 🙂

Reply:
Ok, opened one: https://github.com/swiftlang/swift/issues/76168